### ISSUE TITLE
[ws-manager-bridge] Introduce workspace cluster management API

### ISSUE THREAD

**Issue by csweichel** [gitpod-io/gitpod#3345]
Prerequisite: #3323 

With [pluggable workspace cluster](https://www.notion.so/gitpod/Pluggable-Workspace-Cluster-5180bbdfb3554c758095b1368e50f3ea) ws-manager-bridge gains a fourth responsibility: manage the workspace clusters which are currently connected to the meta/fat cluster. To this end, ws-manager-bridge needs to gain an API through which this functionality is controlled.

The [design doc](https://www.notion.so/gitpod/Pluggable-Workspace-Cluster-5180bbdfb3554c758095b1368e50f3ea#e30553c0d897475bbf9a7e5f9d231801) lays out this API. The purpose of this change/issue is to implement that API and tie it in with the multi-tenancy-feature coming to ws-manager-bridge ( #3344).

Note: actually implementing deregistration is not part of this change, but rather <insert-isssue-here>
